packages/adminrouter/extra/src/README.md

```diff
@@ -870,3 +1112,842 @@ expressions/statements:
 This is why the context of each endpoint is not protected by locks, in
 case when it's only about fetching a single value from context dict or
 storing/appending one there.
+
+## Tests writing howto
+
+This section is intended to quickly enable contributors to start writing tests.
+All the test harness components and concepts are described here with a "bigger
+picture" in mind so that it is easier to understand how they relate to each
+other.
+
+It is divided into three sections:
```

Comment:
`This section (first paragraph)... This section(change you are sugesting) is divided into three subsections(my edit)` - sounds bad IMO, I suggest we stay with `It` here.